src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -1,440 +1,481 @@
-                {
-                    if (argsProvided > 1 ||
-                        argsProvided == 1 && parms.Arguments[0]?.GetType() != typeof(object[]))
-                    {
-                        parms.Arguments = new object[] { parms.Arguments };
-                    }
-                }
-
-                if (argsProvided == argsNeeded)
-                    PerformSpecialConversions(parms.Arguments, parameters);
-            }
-            catch (Exception ex)
-            {
-                parms = new TestCaseParameters(ex);
-            }
-
-            return parms;
-        }
-
-        /// <summary>
-        /// Performs several special conversions allowed by NUnit in order to
-        /// permit arguments with types that cannot be used in the constructor
-        /// of an Attribute such as TestCaseAttribute or to simplify their use.
-        /// </summary>
-        /// <param name="arglist">The arguments to be converted</param>
-        /// <param name="parameters">The ParameterInfo array for the method</param>
-        private static void PerformSpecialConversions(object?[] arglist, IParameterInfo[] parameters)
-        {
-            for (int i = 0; i < arglist.Length; i++)
-            {
-                object? arg = arglist[i];
-                Type targetType = parameters[i].ParameterType;
-                if (ParamAttributeTypeConversions.TryConvert(arg, targetType, out var argAsTargetType))
-                {
-                    arglist[i] = argAsTargetType;
-                }
-            }
-        }
-        #endregion
-
-        #region ITestBuilder Members
-
-        /// <summary>
-        /// Builds a single test from the specified method and context.
-        /// </summary>
-        /// <param name="method">The MethodInfo for which tests are to be constructed.</param>
-        /// <param name="suite">The suite to which the tests will be added.</param>
-        public IEnumerable<TestMethod> BuildFrom(IMethodInfo method, Test? suite)
-        {
-            TestMethod test = new NUnitTestCaseBuilder().BuildTestMethod(method, suite, GetParametersForTestCase(method));
-
-            if (IncludePlatform != null || ExcludePlatform != null)
-            {
-                if (test.RunState == RunState.NotRunnable || test.RunState == RunState.Ignored)
-                {
-                    yield return test;
-                    yield break;
-                }
-
-                var platformHelper = new PlatformHelper();
-
-                if (!platformHelper.IsPlatformSupported(this))
-                {
-                    test.RunState = RunState.Skipped;
-                    test.Properties.Add(PropertyNames.SkipReason, platformHelper.Reason);
-                }
-            }
-
-            yield return test;
-        }
-
-        #endregion
-    }
-}
+// ***********************************************************************
```

Comment:
I originally merged this in the web interface. Not sure what is going on, probably line-feeds. I'll check and update so we get a proper diff.

Reply:
Looking at the history, it looks like you fixed the line endings in this file a couple of months ago @jnm2. I checked and the line endings are all still CRLF, so I'm not sure why it isn't showing a diff. You'll have to take my word that it is the same as your previously reviewed @jnm2.

Reply:
Just tried this and it worked:
![image](https://user-images.githubusercontent.com/8040367/81519893-ada12380-9310-11ea-8c56-31aff1062703.png)